strawberry/aiohttp/handlers/graphql_ws_handler.py

```diff
@@ -1,11 +1,15 @@
 from contextlib import suppress
-from typing import Any, Optional
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Any, Optional
```

Comment:
do you if there's a way to skip having these imports behind `if TYPE_CHECKING`?

feels a bit annoying to have to do that all the time for things inside typing (we still need to import TYPE_CHECKING anyway) :D

Reply:
agree, quite annoying. I'm not sure if there's a way. I'll pause the work and do some digging. Thanks for flagging.

Reply:
https://github.com/charliermarsh/ruff/issues/2267

Reply:
looks like this is addressed in latest release of `ruff` - https://github.com/charliermarsh/ruff/releases/tag/v0.0.237

Reply:
I'll rollback those changes, thanks again for flagging before I went too deep